sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformationExec.scala

```diff
@@ -1,468 +0,0 @@
-/*
```

Comment:
Just as a small tip, you could use `git mv` so it keeps the track when you rename.

Reply:
> Just as a small tip, you could use `git mv` so it keeps the track when you rename.

I use this seems still as before.